### ISSUE TITLE
Error Running GIT

### ISSUE THREAD

**Issue by SamiGhanim** [git-lfs/git-lfs#2682]
Hello developers,
I am new to GitHub and I tried to install windows github on my computer, but whenever i open the GitHub i get this error : 
`
Error running git [config --global filter.lfs.clean git-lfs clean -- %f]: 'error                                                                 : could not lock config file C:/Users/Ghanim/Desktop/Sami/Terminal Coding/.gitco                                                                 nfig: No such file or directory' 'exit status 255'
Run `git lfs install --force` to reset git config.
`
and whenever i run the code it does not work and repeats the error.
also I remember having a cygwin old terminal with that path but I deleted the cygwin application and I can't find the path to delete it, so if you guys can help me to reset my PATH to the default one and fix this problem i would be thankful.

**Comment by technoweenie** [comment]
Looks like your global git config file doesn't exist. You can either create this file: `C:/Users/Ghanim/Desktop/Sami/Terminal Coding/.gitconfig`, or run `git lfs install --system` to update your system config.

**Comment by keith-taylor** [comment]
When I try this I get:
```
❯ git lfs install --system
git: 'lfs' is not a git command. See 'git --help'.

The most similar command is
	log
```

**Comment by chrisd8088** [comment]
Hey @keith-taylor, I'm sorry you're having trouble.  In general, we advise people not to comment on older, closed issues unless it's exactly the same issue.  In this case, I might suggest opening a [discussion](https://github.com/git-lfs/git-lfs/discussions); one can also open a new issue and fill out the issue template with the requested details if you've run into a definite bug.

As it happens, I think your situation is simply that you don't have the Git LFS software installed in a place where Git can find it -- perhaps not installed at all.  I realize that may seem like a tautology because you're running `git lfs install` and that can sound like a request to install Git LFS!

However, when you run `git foo ...` what Git does is search for a binary executable named `git-foo` to run.  In this case, `git lfs ...` asks Git to look for the `git-lfs` binary.  That needs to be in place before any `git lfs ...` commands will work.  So you should follow the appropriate installation steps for your OS.  There are downloads available on the Git LFS [home page](https://git-lfs.github.com/) and if you're on Windows, you might want to use [Git for Windows](https://gitforwindows.org/) which includes Git LFS.  If you're on macOS or Linux, try first with your normal package manager, like Homebrew or `apt`.

Once you have the Git LFS binary installed in a place where Git can find it, then you should be able to run a simple command like `git lfs version`.  If that's working, you can then run `git lfs install`.  As the [manual page](https://github.com/git-lfs/git-lfs/blob/52d47f085ba5ae534dd26ba76d010a537ddbdc8e/docs/man/git-lfs-install.adoc) mentions, this command (which should perhaps have been named something less confusing!) installs the Git filters and hooks so that your system is set up to use Git LFS.

If you use the `--system` option then you're asking for those filter configurations to be placed in `/etc/gitconfig` and will thus affect all users of the system, whereas the default is to put them in the "global" `~/.gitconfig` file where they affect all Git repositories for the current user (hence, "global") but not for all users.  If you're on Linux or macOS and you really want `--system`, you'll probably need `sudo git lfs install --system` since you'll be writing a system-level file.
